### DISCUSSION TITLE
Using the 3D printed case for a miniDEXED

### DISCUSSION THREAD

**Discussion by GregDunn1953** [probonopd/MiniDexed#651] [Q&A] [answered]
First of all, thanks for this amazing project.  I assembled a pi 3B+, serial DAC, encoder and 1602 LCD display; it worked first time.

I had a friend print out the 3D case (provided in the project) for me, and am trying to put the whole thing together.  The Pi and modules/encoder all mount fairly easily.  Unfortunately, the LCD display's I2C backpack/connector protrudes into the area where the DAC board needs to go.  I had to bend the pins on the display to even get it to mount due to clearances.  Also, the Dupont connectors on all the boards are long enough that they intrude on a lot of the interior box volume, making it difficult to fit the case together.  I had to bend the pins on the display to even get it to mount due to clearances.

I have just about decided to directly solder wires to the boards (except the Pi) so everything can be routed and squeezed together safely.  Can anyone share their build using the 3D case showing how you've worked around the restrictions?  It's a real pain to get another case printed where I live, whereas I can try different boards or wiring without much effort.

**Comment by probonopd** [comment, marked as answer]
Yes, it's called "Mini"Dexed for a reason ;-) Indeed by soldering directly to the display PCB (not using headers) and by using Dupont connectors for the headers of the Raspberry Pi I was able to barely squeeze everything in.

But make extra sure that conductive things can't touch each other that shouldn't - I've fried pins on a Raspberry Pi this way.

Here is how mine looks:

![image](https://github.com/probonopd/MiniDexed/assets/2480569/bccaf8e8-4a10-4de5-aeec-b5c15f343c39)

**Reply by GregDunn1953** [reply]
That's essentially my plan.  Glad to see that there's enough room if the wires are routed properly!

**Comment by GregDunn1953** [comment]
![case](https://github.com/probonopd/MiniDexed/assets/48167374/0a4ccc5d-dd44-47a8-88df-c33cd9287d6b)

It works great.  Now all I have to do is edit the default performance.ini file so I just get a single TG for testing...

**Reply by probonopd** [reply]
Great build! I used some black tape on the display PCB to cover the light leakage of the display to the left and right.

**Reply by GregDunn1953** [reply]
I just wanted to make sure everything fit before I started any further customizations.  So far the leakage doesn't particularly bother me, and the unit works great!
